hr_recruitment_security/tests/test_access_rights.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+from odoo.exceptions import AccessError
+from odoo.tests.common import SavepointCase
+
+_logger = logging.getLogger(__name__)
+
+
+class TestHrRecruitmentSecurity(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestHrRecruitmentSecurity, cls).setUpClass()
+        # Users
+        users = cls.env["res.users"].with_context(no_reset_password=True)
+        # hr_officer_own_documents
+        group_name = "group_hr_recruitment_officer_own_documents"
+        cls.user_hr_officer_own_documents = users.create(
```

Comment:
I miss here Recruitment manager to be tested as well.

Reply:
Added now test about Recruitment manager user